core/src/scss/components/global-footer/_global-footer.scss

```diff
@@ -93,6 +93,9 @@
   flex-grow: 1;
 }
 
+.su-global-footer__link--external {
```

Comment:
BEM naming convention is off. This is a whole new element and not a modifier of the link element. Perhaps `su-global-footer__link-a11y`?